packages/fast-components-react-msft/src/carousel/carousel.tsx

```diff
@@ -223,6 +225,33 @@ class Carousel extends Foundation<
         return this.slides.length !== 1;
     }
 
+    /**
+     * Check if it is the FIRST slide for looping
+     */
+    private get isFirstSlide(): boolean {
+        if (this.props.items.length) {
+            const firstSlideId: string = this.props.items[0].id;
+            if (firstSlideId) {
```

Comment:
Do you need this `if (firstSlideId)` check? Presumably if this was falsey, `firstSlideId === this.state.activeId` would also be falsey

Reply:
true, i am already checking to make sure the length exists. making this change passes the tests too so I'll add it.